diff --git a/dist/plugins/email-otp/index.cjs b/dist/plugins/email-otp/index.cjs
index e7410365b62ed896582412f6901f2df89d7d9570..e8a2288fd3e3bb9477113ba04d496c2c42da7889 100644
--- a/dist/plugins/email-otp/index.cjs
+++ b/dist/plugins/email-otp/index.cjs
@@ -613,6 +613,12 @@ const emailOTP = (options) => {
               message: ERROR_CODES.INVALID_OTP
             });
           }
+          if (ctx.body.password === '') {
+            throw new betterCall.APIError("BAD_REQUEST", {
+              message: 'INVALID_PASSWORD'
+            });
+          }
+  
           await ctx.context.internalAdapter.deleteVerificationValue(
             verificationValue.id
           );
diff --git a/dist/plugins/email-otp/index.mjs b/dist/plugins/email-otp/index.mjs
index 4aa4efcabb5365e6215118739c33b712c6790d43..dcd686f4d01a20e5dc9e20fb5d609fbf8cdd7eb5 100644
--- a/dist/plugins/email-otp/index.mjs
+++ b/dist/plugins/email-otp/index.mjs
@@ -611,6 +611,11 @@ const emailOTP = (options) => {
               message: ERROR_CODES.INVALID_OTP
             });
           }
+          if (ctx.body.password === '') {
+            throw new APIError("BAD_REQUEST", {
+              message: 'INVALID_PASSWORD'
+            });
+          }
           await ctx.context.internalAdapter.deleteVerificationValue(
             verificationValue.id
           );
